src/omero/plugins/duplicate.py

```diff
@@ -51,7 +52,23 @@
     # that would have been duplicated
     omero duplicate Dataset:53 --dry-run --report
 
+    # Duplicate a project with its datasets but not their images
+    omero duplicate Project:15 --ignore=DatasetImageLink
+    # Duplicate a project with the original images linked from its datasets
+    omero duplicate Project:15 --reference=Image
 """
+        "    # Duplicate a project, linking to the original annotations "
+        "except for duplicating the comments and ratings\n"
+        "    omero duplicate Project:15 --reference=Annotation "
+        "--duplicate=CommentAnnotation,LongAnnotation\n"
+        """
+Group permissions can prevent a duplicated link from referencing another
+user's Image or Annotation. However, note that ignoring a linked-to
```

Comment:
i.e. it will throw an exception?

Reply:
@joshmoore Yes, see https://github.com/ome/omero-cli-duplicate/pull/12#issuecomment-690347660

Reply:
Sorry, I'm pointing out we may want to say that to the user.